pd/src/verify.rs

```diff
@@ -39,8 +39,8 @@ pub struct PendingTransaction {
     pub spent_nullifiers: BTreeSet<Nullifier>,
     /// Delegations performed in this transaction.
     pub delegations: Vec<Delegate>,
-    /// Undelegations performed in this transaction.
-    pub undelegations: Vec<Undelegate>,
+    /// Undelegation, if any, performed in this transaction (there must be no more than one).
+    pub undelegation: Option<Undelegate>,
```

Comment:
We now have a consensus rule saying there's no more than one undelegation per transaction, to simplify user comprehension and our own implementation.

Reply:
Question: is the consensus rule that there is *no more than one undelegation per transaction*, or that there is *no more than one validator undelegated from*? i.e., is it allowed to have multiple `Undelegate` descriptions for the same validator?

(I don't think the choice matters, since there's no reason to have multiple, but just want to be precise about it).

Reply:
NVM, resolved below.